go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/docker/buildx
 
-go 1.23.0
+go 1.23.1
```

Comment:
It should be 1.23.0. Seems related to https://github.com/tonistiigi/jaeger-ui-rest/blob/bb2801509720cbedaac29fbefeeef9ba24e54c45/go.mod#L3

Looking at code on this repo I don't think there is any requirement for this specific patch release.

Reply:
Oh would need 1.22.0 actually looking at https://github.com/docker/buildx/pull/2978/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R3

Reply:
Opened https://github.com/tonistiigi/jaeger-ui-rest/pull/1